### PR TITLE
Use the common documentation from `google/open-source-pdks` repository

### LINKED ISSUE #23: Add documentation about versioning system
The documentation should include information about how the PDK is versioned.

See related issue at https://github.com/google/open-source-pdks/issues/8

### PR DESCRIPTION
**Pull request by mithro**
The repository at [`google/open-source-pdks`](https://github.com/google/open-source-pdks) contains common information which applies to all of the Google maintained open source PDKs.

This includes;
 * File naming information - https://github.com/google/open-source-pdks/issues/7
 * Versioning scheme -  https://github.com/google/open-source-pdks/issues/8
 * Code of Conduct details
 * Contributing guidelines
 * Other open source PDKs that exist.

Fixes https://github.com/google/gf180mcu-pdk/issues/23 and https://github.com/google/gf180mcu-pdk/issues/22.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+This project follows the [Google Open Source Community Guidelines](https://opensource.google/conduct).
```
Comment (proppy) [inline CODE_OF_CONDUCT.md]: Does this need to change to markdown? (seems unrelated to the PR current description)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Guidelines for contributing can be found locally in this repository at
```
Comment (proppy) [inline CONTRIBUTING.md]: Does this need to change to markdown? (seems unrelated to the PR current description)

**Comment by proppy** [comment]
`Versioning Information` seems to appear two times with this change:
Screenshot_20220726-004234

**Comment by mithro** [comment]
> `Versioning Information` seems to appear two times with this change:

Fixed.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (mithro) [reply CODE_OF_CONDUCT.md]: I thought markdown would be more readable, but actually RST does look nicer.

File: CONTRIBUTING.md (hunk 2 above)
Comment (mithro) [reply CONTRIBUTING.md]: I thought markdown would be more readable, but actually RST does look nicer.
